src/blister.h

```diff
@@ -352,6 +357,148 @@ extern void ConvBlist (
 
 /****************************************************************************
 **
+*F  CopyBits( <fromblock>, <from-starting-bit>, <toblock>, <to-starting-bit>,
+**            <numbits> )
+**
+**  `CopyBits' copies <numbits> bits (numbering bits within a UInt
+**   from the least significant to the most significant) starting with
+**   bit number <from-starting-bit> of UInt *<fromblock> to a destination
+**   starting at bit <to-starting-bit> of *<toblock>.
```

Comment:
Are the two blocks assumed to be non-overlapping? I think it would be a good idea to document this explicitly.